README.md

```diff
@@ -26,64 +29,70 @@ Basic Concept
 Getting Started
 ---------------
 
-- You need Python3. We tested primarily with **python 3.5**
-- This was tested with a recent Ubuntu based Linux.
-
+- You need Python3.6 and above for installing tanner.
+- This was tested with a recent Ubuntu-based Linux.
 
-### Setup Redis
+### Steps to install TANNER
 
+#### Step 1: Setup Redis
 
 1. Install the Redis: ``sudo apt-get install redis-server``
-2. Start it on ``localhost`` with default ``port``: ``redis-server``
-
-### Setup PHP Sandbox
+2. Run ``redis-server`` (to start it on `localhost` with default `port`)
 
+#### Step 2: Setup PHP Sandbox
 
-1. For PHP Sandbox setup, see sandbox [manual](https://github.com/mushorg/phpox)
-2. In PHP Sandbox directory, run sandbox: ``sudo python3 sandbox.py``
+1. Clone the repo using:  ``git clone https://github.com/mushorg/phpox.git`` (preferably in your home directory).
+2. Go to PHP Sandbox directory: ``cd phpox``
+2. Run sandbox: ``sudo python3 sandbox.py``
```

Comment:
phpox should be setup before running ``sudo python3 sandbox.py``, so the first step in old version is really the first

Reply:
Sorry, my bad. There is only `make` command to setup phlox according to the README; I was going to add that.

Apart from that, there isn't anything needed to setup phpox, I think. So shifted the manual link to the bottom.